test/application_test.rb

```diff
@@ -71,4 +71,19 @@ def self.loaded?
       @application.name.must_equal @application.class.name
     end
   end
+
+  describe 'Logger' do
+    it 'has logger constant and respond to info method' do
+      assert_respond_to(CoffeeShop::Application::Logger, :info?)
+    end
+
+    it 'log application namespace' do
+      output =
+        stub_stdout_constant do
+          class Cocacola < Lotus::Application; end
+          Cocacola::Logger.info('')
+        end
+      assert_match(/Cocacola/, output)
```

Comment:
@joneslee85 if there is no special reason, i would like to keep using assert style.